plugins/CoreConsole/Commands/DevelopmentSyncUITestScreenshots.php

```diff
@@ -73,42 +71,166 @@ protected function execute(InputInterface $input, OutputInterface $output)
             if ($file !== null
                 && preg_match("/" . $screenshotsRegex . "/", $file)
             ) {
-                if ($testPlugin == null) {
-                    $downloadTo = "tests/UI/expected-ui-screenshots/$file";
-                } else {
-                    $downloadTo = "plugins/$testPlugin/tests/UI/expected-ui-screenshots/$file";
-                }
-
-                $output->write("<info>Downloading $file to  $downloadTo...</info>\n");
-                Http::sendHttpRequest("$urlBase/processed-ui-screenshots/$file", $timeout = 60, $userAgent = null,
-                    PIWIK_DOCUMENT_ROOT . "/" . $downloadTo);
+                $this->downloadProcessedScreenshot($input, $output, $file);
             }
         }
 
-        $this->displayGitInstructions($output);
+        $this->displayGitInstructions($input, $output);
 
     }
 
-    /**
-     * @param OutputInterface $output
-     */
-    protected function displayGitInstructions(OutputInterface $output)
+    protected function displayGitInstructions(InputInterface $input, OutputInterface $output)
     {
         $output->writeln('');
         $output->writeln('--------------');
         $output->writeln('');
         $output->writeln("If all downloaded screenshots are valid you may push them with these commands:");
-        $output->writeln('');
-        $commands = "cd tests/UI/expected-ui-screenshots
+        $downloadToPath = $this->getDownloadToPath($input);
+        $commands = "
+cd $downloadToPath
 git pull
 git add .
-git commit -m '' # WRITE A COMMIT MESSAGE
-git push
+git commit -m '' # Write a good commit message
+git push";
+
+        $plugin = $input->getOption('plugin');
```

Comment:
I see this specific $input option access, it might be better to keep it one place, what do you think?